recipes/renode-cores/variants.yaml

```diff
@@ -0,0 +1,12 @@
+c_stdlib:
+    - if: linux
+      then: sysroot
+    - if: osx
+      then: macosx_deployment_target
+
+m2w64_c_compiler:
+  - m2w64-toolchain
+m2w64_cxx_compiler:
+  - m2w64-toolchain
+m2w64_c_stdlib:
+  - m2w64-toolchain
```

Comment:
```suggestion
```
You shouldn't need this file in `conda-forge` - these defaults are already supposed to set in both staged-recipes and the feedstock.  You might need it to build locally, but it shouldn't be included in the recipe submission